### DISCUSSION TITLE
Web3-validator

### DISCUSSION THREAD

**Discussion by meri-maki** [ethereum/web3.js#6321] [Q&A]
How do you use web3-validator correctly?

Trying to use isAddress function to check user input for contract voting. Downloaded ```web3-validator``` --> Called ```web3.validator.isAddress(address)``` --> Get error ```Uncaught TypeError: Cannot read properties of null (reading 'validator')```

Am I missing something?

**Comment by TatyOko28** [comment]
The error "Uncaught TypeError: Cannot read properties of null (reading 'validator')" means that web3.validator does not exist in your Web3.js instance.

 Fix: web3-validator is not part of Web3.js. Instead, use the correct package or web3.utils.isAddress() for validation.

**Reply by FiskPay** [reply]
Hello! The solution to this issue is:

```
const { Web3, validator } = require("web3");
.......
IsAddress = (address) => {

        return validator.isAddress(address);
    }
```
